private_set_intersection/cpp/BUILD

```diff
@@ -73,17 +72,17 @@ cc_test(
     linkopts = PSI_LINKOPTS,
     deps = [
         ":bloom_filter",
-        "//psi_cardinality/cpp/util:status_matchers",
+        "//private_set_intersection/cpp/util:status_matchers",
         "@com_google_absl//absl/strings",
         "@com_google_googletest//:gtest",
         "@com_google_googletest//:gtest_main",
     ],
 )
 
 cc_library(
-    name = "psi_cardinality_client",
-    srcs = ["psi_cardinality_client.cpp"],
-    hdrs = ["psi_cardinality_client.h"],
+    name = "psi_client",
+    srcs = ["psi_client.cpp"],
+    hdrs = ["psi_client.h"],
     includes = ["."],
     deps = [
         ":bloom_filter",
```

Comment:
Should we mark bloom_filter with visibility:private?

Reply:
Done.